utils/run_in_ga.sh

```diff
@@ -14,6 +14,10 @@ rm -rf /opt/daos/prereq/release/spdk
 $SCONS PREFIX=/opt/daos --build-deps=yes --deps-only
 echo ::endgroup::
 
+echo ::group::Stack analyzer output
+$SCONS --jobs 10 --analyze-stack="-x tests -c 128"
```

Comment:
This could possibly just run on the server which would help the build-time slightly.
```suggestion
$SCONS --jobs 10 --analyze-stack="-x tests -c 128" server
```